docs/api/saved-objects/copy_to_space.asciidoc

```diff
@@ -0,0 +1,290 @@
+* Include a range of valid values, if applicable.
+* If the parameter requires a specific delimiter for multiple values, say so.
+* If the parameter supports wildcards, ditto.
+* For large or nested objects, consider linking to a separate definition list.
+***************************************
+////
+
+
+[[saved-objects-api-copy-to-space-path-params]]
+==== {api-path-parms-title}
+////
+A list of all the parameters within the path of the endpoint (before the query string (?)).
+
+For example:
+`<follower_index>`::
+(Required, string) Name of the follower index
+////
+`space_id`::
+(Optional, string) Identifies the source space from which saved objects will be copied. If `space_id` is not specified in the URL, the default space is used.
+
+//// 
+[[saved-objects-api-copy-to-space-params]]
+==== {api-query-parms-title}
+////
+////
+A list of the parameters in the query string of the endpoint (after the ?).
+
+For example:
+`wait_for_active_shards`::
+(Optional, integer) Specifies the number of shards to wait on being active before
+responding. A shard must be restored from the leader index being active.
+Restoring a follower shard requires transferring all the remote Lucene segment
+files to the follower index. The default is `0`, which means waiting on none of
+the shards to be active.
+////
+
+[[saved-objects-api-copy-to-space-request-body]]
+==== {api-request-body-title}
+////
+A list of the properties you can specify in the body of the request.
+
+For example:
+`remote_cluster`::
+(Required, string) The <<modules-remote-clusters,remote cluster>> that contains
+the leader index.
+
+`leader_index`::
+(Required, string) The name of the index in the leader cluster to follow.
+////
+`spaces` ::
+  (Required, string array) The ids of the spaces the specified object(s) will be copied into.
+
+`objects` ::
+  (Required, object array) The saved objects to copy.
+  `type` :::
+    (Required, string) The saved object type.
+  `id` :::
+    (Required, string) The saved object id.
+
+`includeReferences` ::
+  (Optional, boolean) When set to `true`, all saved objects related to the specified saved objects will also be copied into the target spaces. The default value is `false`.
+
+`overwrite` ::
+  (Optional, boolean) When set to `true`, all conflicts will be automatically overidden. If a saved object with a matching `type` and `id` exists in the target space, then that version will be replaced with the version from the source space. The default value is `false`.
+
+
+[[saved-objects-api-copy-to-space-response-body]]
+==== {api-response-body-title}
+////
+Response body is only required for detailed responses.
+
+For example:
+`auto_follow_stats`::
+  (object) An object representing stats for the auto-follow coordinator. This
+  object consists of the following fields:
+
+`auto_follow_stats.number_of_successful_follow_indices`:::
+  (long) the number of indices that the auto-follow coordinator successfully
+  followed
+...
+
+////
+
+`<space_id>`::
```

Comment:
Trying to convey that the response object will have dynamic keys, based on the space ids specified in the request body.

Reply:
How about:

Specifies the dynamic keys that are included in the response.